pypeit/pypeit.py

```diff
@@ -457,7 +457,15 @@ def reduce_exposure(self, frames, bg_frames=None, std_outfile=None):
         all_spec2d['meta']['find_negative'] = self.find_negative
         # TODO -- Should we reset/regenerate self.slits.mask for a new exposure
 
+        # container for specobjs during first loop (objfind)
         all_specobjs = specobjs.SpecObjs()
+        # container for specobjs during second loop (extraction)
+        all_specobjs_new = specobjs.SpecObjs()
```

Comment:
Call this something more informative, like merged or union or something like that to indicate it merges all detectors.

Reply:
Done!